extensions/wikia/AdEngine/js/provider/directGptMobile.js

```diff
@@ -16,6 +17,25 @@ define('ext.wikia.adEngine.provider.directGptMobile', [
 			MOBILE_IN_CONTENT_EXTRA_2:  {size: '300x250,1x1'},
 			MOBILE_IN_CONTENT_EXTRA_3:  {size: '300x250,1x1'},
 			MOBILE_PREFOOTER:           {size: '320x50,300x250,300x50,1x1'}
+		},
+		{
+			beforeSuccess: function (slotName) {
+				btfBlocker.onSlotResponse(slotName);
+			},
+			beforeCollapse: function (slotName) {
+				btfBlocker.onSlotResponse(slotName);
+			},
+			beforeHop: function (slotName) {
+				btfBlocker.onSlotResponse(slotName);
+			}
 		}
 	);
+
+	btfBlocker.init('mercury', provider.fillInSlot);
```

Comment:
1. What do you think about moving this part to `factoryWikiaGpt.js`?
2. What do you think about passing config (list of slots) instead of skin?